lib/scope.js

```diff
@@ -44,6 +45,10 @@ class RestifizerScope {
     return this.checkActionName(ACTIONS.SELECT, ACTIONS.SELECT_ONE, ACTIONS.COUNT);
   }
 
+  isSelectDistinct() {
+    return this.checkActionName(ACTIONS.DISTINCT);
```

Comment:
It's a select by the meaning, and you named the method `isSelectDistinct` approving this. If you have an issue in your project after this change, it's better to update the code in your project.

Just in case, I mean to add DISTINCT checking to isSelect, and keep this method as well.